### ISSUE TITLE
gcloud cli examples for service account creation

### ISSUE THREAD

**Issue by andrewhatfield** [NetAppDocs/astra#111]
Page: [Set up Google Cloud](https://docs.netapp.com/us-en/astra/get-started/set-up-google-cloud.html)

To accelerate user adoption and reduce friction, it would be great to have gcloud api examples

```gcloud iam service-accounts create SERVICE_ACCOUNT_ID \
    --description="DESCRIPTION" \
    --display-name="DISPLAY_NAME"```

**Comment by netapp-mwallis** [comment]
Hi Andrew, thanks for this suggestion.  I'll work with product management to include some useful examples.  Also, is this issue a duplicate of https://github.com/NetAppDocs/astra-control-service/issues/110?

**Comment by andrewhatfield** [comment]
#111 and #110 are different issues

**Comment by netapp-mwallis** [comment]
@andrewhatfield, the GCloud setup documentation [here ](https://docs.netapp.com/us-en/astra/get-started/set-up-google-cloud.html) does link to Google documentation, which has API examples for service account creation:
https://cloud.google.com/iam/docs/creating-managing-service-accounts#iam-service-accounts-create-rest

Would it help to make it more clear that this link has API examples in the link text itself?

**Comment by andrewhatfield** [comment]
Yes, make it easy for people without forcing them to jump through hoops.

Give them complete copy and paste instructions including CLI and code

**Comment by netapp-mwallis** [comment]
Hi @andrewhatfield, understood.  Unfortunately I was informed that we have a policy against copying content from other vendors, for several reasons:
- We have no control over the content, and it could change at any time and be out of date
- We don't have permission from Google to reuse their documentation, images, or script / code examples

Sorry about that.  We'll have to settle for linking to the content, but I will make it clearer that the destination link includes API examples as well.

**Comment by andrewhatfield** [comment]
I'm not suggesting we copy documentation, I'm saying that we should provide
all of the necessary steps for our user to complete the tasks required.

Sending a customer to another website to trawl through documentation to
work it out themselves is not user friendly.

Just as we provide instructions on how to click through a UI (which changes
without our knowledge creating a burden on the user, support, and
documentation) we should equally provide the CLI instructions in a clear
and concise manner.
